Rsk.Samples.IdentityServer.AdminUiIntegration/Services/CustomEventSink.cs

```diff
@@ -24,13 +24,20 @@ public CustomEventSink(ILogger<DefaultEventService> logger, IEventStore eventSto
 
         public Task PersistAsync(Event evt)
         {
-            if (evt == null) throw new ArgumentNullException(nameof(evt));
+            try
+            {
+                if (evt == null) throw new ArgumentNullException(nameof(evt));
+
+                logger.LogInformation("{@event}", evt);
 
-            logger.LogInformation("{@event}", evt);
-            
-            if (evt.EventType == EventTypes.Error || evt.EventType == EventTypes.Failure) eventStore.AddEvent(evt); 
-            
-            return Task.CompletedTask;
+                if (evt.EventType == EventTypes.Error || evt.EventType == EventTypes.Failure) eventStore.AddEvent(evt);
+
+                return Task.CompletedTask;
+            }
+            catch(Exception)
+            {
+                return Task.CompletedTask;
```

Comment:
If we suppress all exceptions here, we should at least log something to alert people to something going wrong.

Reply:
Added log.